.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,25 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+**Environment:**
```

Comment:
Thanks for adding the template. Can we also add:

7. Python version:
8. OS and version: